services/movie_session.py

```diff
@@ -0,0 +1,42 @@
+from db.models import MovieSession
+from django.db.models import QuerySet
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int
+                         ) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.filter(show_time__date=session_date)
+    else:
+        return MovieSession.objects.all()
```

Comment:
You can simplify this code
```suggestion
    movie_session =  MovieSession.objects.all()
    if session_date:
        movie_session = movie_session.objects.filter(show_time__date=session_date)
    return movie_session
```